src/main/java/gov/usgs/cida/pubs/busservice/ipds/IpdsParserService.java

```diff
@@ -57,15 +59,17 @@ protected Integer getFirstNodeInteger(final Element element, final String tagNam
 		return rtn;
 	}
 
-	protected String formatOrcid(String orcid) {
+	public static String formatOrcid(String orcid) {
```

Comment:
Our respective code changes are are going to collide here. The entire *Ipds* related code is being removed. I like the idea of a static function to format the orcid - perhaps PubsUtils would be a good place to put it. I think we also talked about just storing the unique portion of the orcid, rather than the entire url. I was going to use something like:

Pattern ORCID_PATTERN = Pattern.compile("\\d{4}-\\d{4}-\\d{4}-(\\d{3}X|\\d{4})");
Matcher matcher = ORCID_PATTERN.matcher(orcid);
if (matcher.find()) {
	formattedOrcid = matcher.group();
} else {
	formattedOrcid = null;
}